osgp/shared/osgp-ws-smartmetering/src/main/resources/SmartMeteringMonitoring.wsdl

```diff
@@ -109,6 +109,8 @@
     <wsdl:part element="smmon:ActualMeterReadsGasResponse"
       name="ActualMeterReadsGasResponse" />
   </wsdl:message>
+  
+  
```

Comment:
Avoid multiple blank lines, and don't leave trailing spaces.

Reply:
removed trailing spaces and multiple blank lines